.github/actions/latest-stable-tag/dist/index.js

```diff
@@ -0,0 +1,8334 @@
+import { createRequire as __WEBPACK_EXTERNAL_createRequire } from "module";
+/******/ var __webpack_modules__ = ({
```

Comment:
WHAT IS GOING ON HERE?
There are so many /******/s. and 7199s and cryptic variables.
This.... this must be generated. It's crazy

Maybe you made a note about it. I'll keep looking.

Reply:
Yes, it's generated. I followed the steps in [this page](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github). The alternative was to commit the `node_modules` folder, which has more than 5000 files. (one of the things I hate about the JS ecosystem...)

Reply:
The `node_modules` folder is a blight!